agent/xds/rbac.go

```diff
@@ -237,9 +263,20 @@ func intentionActionFromString(s structs.IntentionAction) intentionAction {
 	return intentionActionDeny
 }
 
+type rbacService struct {
+	structs.ServiceName
+
+	// peer, exportedPartition, and trustDomain are
+	// only applicable to imported services and are
+	// used to override SPIFFEID fields.
+	peer              string
+	exportedPartition string
+	trustDomain       string
```

Comment:
```suggestion
	Peer              string
	ExportedPartition string
	TrustDomain       string
```

Given that you embedded `structs.ServiceName` this means that one of your 4 fields is exported and the others are not leading to some odd feeling struct declarations with mixed case.

This `rbacService` is an entirely internal construction which itself is not exported so it wouldn't hurt to just "export" the fields for consistency.